packages/asset-swapper/contracts/src/BancorV3Sampler.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2022 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+
+import "./interfaces/IBancorV3.sol";
+
+
+contract BancorV3Sampler
+{
+    /// @dev Gas limit for BancorV3 calls.
+    uint256 constant private BancorV3_CALL_GAS = 1500e3; // 150k
```

Comment:
another 0 here?

1500e3 = 1.5m?

Reply:
good catch this was for testing last night